subql/packages/bridger/project/pangolin.yaml

```diff
@@ -42,3 +42,25 @@ dataSources:
           kind: substrate/EventHandler
           filter:
             module: bridgePangoroDispatch
+
+        ## bridge ethv2
```

Comment:
If bridge ethv2 migrate to pangolin, I think we can remove it from pangoro https://github.com/darwinia-network/indexer/blob/f50c9a17956bf7c9fa68399eb8b7ea92d9eaaa1d/subql/packages/bridger/project/pangoro.yaml#L46-L66